C#/cs01/01.cs

```diff
@@ -11,14 +11,36 @@ Fizz ...
     public class FizzBuzzPrinter
     {
         ///Accept one number and return a string(Fizz or Buzz or the number ) based on the aforementioned logic 
-        public bool Print(int num)
+        public string Print(int a)
         {
-            throw new NotImplementedException("Please write your solution here");
+            if(a%3 == 0){
+                return "Fizz";
+            }
+            else if(a% 5== 0){
+                return "Buzz";
+            }
+            else{
```

Comment:
you have to write print function in a such a way that it does not need to be called unncessary. this else is not needed. And you have to write the logic in such  a way that Print needs to be reused.